test/integration/testConnection.js

```diff
@@ -811,3 +813,81 @@ describe('Connection Test - isValid', () => {
 
   // there is no way to test heartbeat fail to running instance of snowflake
 });
+
+it('Connect async with keep alive interval created', async function () {
+  const connection = snowflake.createConnection(connOption.validWithKeepAlive);
+
+  await testUtil.connectAsync(connection);
+  assert.ok(connection.isUp(), 'not active');
+  await testUtil.destroyConnectionAsync(connection);
+  assert.ok(!connection.isUp(), 'still active');
+});
+
+it('Connect async with original callback', async function () {
+  const connection = snowflake.createConnection(connOption.valid);
+
+  await testUtil.connectAsyncWithOriginalCallback(connection, () => {});
+  assert.ok(connection.isUp(), 'not active');
```

Comment:
can we create methods like `assertConnectionActive(connection)` to not duplicate assert messages?